unidist/core/backends/mpi/core/controller/api.py

```diff
@@ -18,7 +18,8 @@
         "Missing dependency 'mpi4py'. Use pip or conda to install it."
     ) from None
 
-from unidist.core.backends.mpi.core.controller.object_store import object_store
+from unidist.core.backends.mpi.core.shared_store import SharedStore
+from unidist.core.backends.mpi.core.object_store import ObjectStore
```

Comment:
I think we should do the following renaming.

SharedStore -> SharedObjectStore
ObjectStore -> LocalObjectStore